docs/pages/versions/unversioned/sdk/image.mdx

```diff
@@ -74,3 +75,59 @@ import { Image } from 'expo-image';
+
+<Terminal cmd={['$ npm install multer sharp blurhash']} />
+
+Next, we need to import all needed functions from installed packages and initialized `multer`:
+
+```js
+// Multer is a middleware for handling `multipart/form-data`.
+const multer = require('multer');
+// Sharp allows you to recieve a data buffer from the uploaded image.
+const sharp = require('sharp');
+// Import the encode function from the blurhash package.
+const { encode } = require('blurhash');
+
+// Initilize `multer`.
+const upload = multer();
+```
+
+Assuming app is a variable that holds our express server, we can create an endpoint that accepts an image and returns a JSON response containing the generated blurhash.
+
+```js
+app.post('/blurhash', upload.single('image'), async (req, res) => {
+  const { file } = req;
+  // If the file is not available we're returning with error.
+  if (file === null) {
+    res.status(400).json({ message: 'Image is missing' });
+    return;
+  }
+
+  // Users can specify number of components in each axies.
+  const compoenentX = req.body.componentX ?? 4;
+  const compoenentY = req.body.componentY ?? 3;
+
+  // We're converting provided image to a byte buffer.
+  // Sharp currencurrentlylty supported multiple common formats like JPEG, PNG, WebP, GIF, and AVIF.
+  const { data, info } = await sharp(file.buffer).ensureAlpha().raw().toBuffer({
+    resolveWithObject: true,
+  });
+
+  const blurhash = encode(data, info.width, info.height, compoenentX, compoenentY);
+  res.json({ blurhash });
+});
+```
+
+Additionally, the request can include two parameters: `componentX` and `componentY` which are passed through the algorithm. These values can be calculated or hard-coded on the server, or specified by the user. However, they must be within the range of 1 to 9 and have an aspect ratio similar to the uploaded image. A value of 9 will give the best results but may take longer to generate the hash.
+
+The process of generating a blurhash can be accomplished in various languages and server technologies, similar to the one using JavaScript. The key step is to locate an encoder for your chosen language, which can often be found in the [`woltapp/blurhash`](https://github.com/woltapp/blurhash#implementations) repository. Once you have the encoder, you will need to obtain a representation of the image. Some libraries use a default image class (for example, the Swift implementation uses `UIImage`), but in other cases, you will have to provide raw byte data. Therefore, be sure to check the encoder readme to confirm the expected data format.
+
+> Note: when working with raw byte data, it is crucial to ensure that all pixels are represented by 4 numbers - the alpha layer is present. Failing to do so may lead to errors like "width and height must match the pixels array".
```

Comment:
```suggestion
> When working with raw byte data, it is crucial to ensure that all pixels are represented by 4 numbers - the alpha layer is present. Failing to do so may lead to errors such as "width and height must match the pixels array".
```

This phrase: "the alpha layer is present", doesn't make sense to me. Is the pixel composition represented by 4 numbers is called "the alpha layer"? If yes, we might want to rephrase the sentence and/or add a link, if possible, from the blurhash docs.